include/lsst/meas/base/PeakLikelihoodFlux.h

```diff
@@ -38,16 +38,16 @@ namespace meas {
 namespace base {
 
 /**
- *  @brief C++ control object for peak likelihood flux.
+ *  @brief C++ control object for peak likelihood instFlux.
  *
- * Peak likelihood flux requires an image that has been filtered by convolving with its own PSF
- * (or an approximate model). It is equivalent to a PSF flux (e.g. PsfFluxAlgorithm) computed on
+ * Peak likelihood instFlux requires an image that has been filtered by convolving with its own PSF
+ * (or an approximate model). It is equivalent to a PSF instFlux (e.g. PsfFluxAlgorithm) computed on
```

Comment:
"It is equivalent to a PSF flux (e.g. PsfFluxAlgorithm)" is pre-existing but I find the "e.g." very confusing. I think the author meant "e.g. as computed by PsfFluxAlgorithm" and if agree and you are willing, please update accordingly. If not, or if in doubt, then please consider ditching the parenthetical comment.